src/eppo_value.ts

```diff
@@ -46,8 +74,14 @@ export class EppoValue {
         return typeof this.numericValue === 'number';
       case ValueType.StringType:
         return typeof this.stringValue === 'string';
-      default:
-        return true;
+      case ValueType.JSONType:
+        return (
+          typeof this.objectValue === 'object' &&
+          typeof this.stringValue === 'string' &&
+          JSON.stringify(JSON.parse(this.stringValue)) === JSON.stringify(this.objectValue)
```

Comment:
Nice JSON type check!

Should we wrap it in a `try`-`catch` in case for some reason invalid JSON works it way to the RAC string value resulting in a thrown error?

Reply:
Yeah good point! Looks like this is the list of exceptions I'd need to handle here:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#exceptions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#exceptions